src/components/metrics/ResourceMetrics.jsx

```diff
@@ -0,0 +1,35 @@
+// Copyright 2019 Stanford University see LICENSE for license
+
+import React from "react"
+import PropTypes from "prop-types"
+import MetricsWrapper from "./MetricsWrapper"
+import CountCard from "./CountCard"
+import useMetric from "hooks/useMetric"
+
+const ResourceMetrics = ({ triggerHandleOffsetMenu }) => {
+  const resourceCountMetric = useMetric("getResourceCount")
+
+  return (
+    <MetricsWrapper
+      triggerHandleOffsetMenu={triggerHandleOffsetMenu}
+      title="Resource metrics"
+    >
+      <div className="row">
+        <div className="col-md-3">
+          {resourceCountMetric?.count && (
+            <CountCard
+              count={resourceCountMetric?.count}
+              title="Resource count"
+            />
+          )}
+        </div>
+      </div>
+    </MetricsWrapper>
+  )
+}
+
+ResourceMetrics.propTypes = {
+  triggerHandleOffsetMenu: PropTypes.func,
+}
+
+export default ResourceMetrics
```

Comment:
I wonder if it makes sense for the component to take an argument as to which kind of metric is desired, since these are all so similar.   I also am fine with it as is;  I think the 4th one is the tipping point, not the 3rd.

Reply:
These will diverge as more metrics are added. It just so happens that the first 3 easiest metrics were all counts.

Reply:
And, this is going to be an evolving interface, so I suspect that there will be more opportunities for refactoring emerging.